python/setup.py

```diff
@@ -217,6 +226,8 @@ def get_package_data_files():
     name="tvm",
     version=__version__,
     description="TVM: An End to End Tensor IR/DSL Stack for Deep Learning Systems",
+    long_description=long_description_contents(),
+    long_description_content_type="text/markdown",
```

Comment:
It's not strictly related but can we round out some other areas for PyPi while we're in here?
```suggestion
    long_description_content_type="text/markdown",
    url='https://tvm.apache.org/',
    download_url='https://github.com/apache/tvm/tags',
    author='Apache TVM',
    license='Apache',
    # See https://pypi.org/classifiers/
    classifiers=[
        "License :: OSI Approved :: Apache Software License",
        "Development Status :: 4 - Beta",
        "Intended Audience :: Developers",
        "Intended Audience :: Education",
        "Intended Audience :: Science/Research",
    ],
    keywords='machine learning',
```

Reply:
Applied, thanks.